packages/ds-medicare-gov/src/styles/settings/variables/_layout.scss

```diff
@@ -1 +1,3 @@
+/* stylelint-disable declaration-no-important -- Temporarily ignore until mgov can be refactored */
 $card-border-radius: 6px !important;
```

Comment:
Maybe this needed to be migrated to tokens? Needs ticket to do deeper investigation.